ledger-tool/src/main.rs

```diff
@@ -1381,6 +1443,27 @@ fn main() {
                 .about("Output statistics in JSON format about \
                         all column families in the ledger rocksdb")
         )
+        .subcommand(
+            SubCommand::with_name("compute-cost")
+            .about("runs cost_model over the block at the given slots, \
+                   computes how expensive a block was based on cost_model")
+            .arg(
+                Arg::with_name("slots")
+                    .long("slots")
```

Comment:
How about we make the arguments positional, so you can run:

1. `solana-ledger-tool compute-cost slot1 slot2 slot3` : limit to these slots
or
2. `solana-ledger-tool compute-cost` : imples all slots

Reply:
yep, that'd make syntax clean.